### PR TITLE
feat: add web driver setup details for flow-client 

### PR DESCRIPTION
**Pull request by taefi**
This update clarifies the need to have to have a chrome web driver installed and how to configure it properly in `flow-client/intern.json` for running the `flow-client` unit tests. It also update the formatting of maven commands for better visibility in docs/website.

### CODE REVIEW

File: articles/contributing/flow-unit-testing.asciidoc [hunk 1]
```diff
@@ -4,11 +4,10 @@ order: 40
...
 ---
 
+= Unit Testing in Flow
 :experimental:
 :commandkey: &#8984;
 
-= Unit Testing in Flow
-
 Every change in the code base requires a JUnit test for the code change.
 In cases where a JUnit test is not plausible then an <<flow-integration-testing#,Integration Test>> should be added instead.
```
Comment (dougjewsbury) [inline articles/contributing/flow-unit-testing.asciidoc:12]:
```suggestion
In cases where a JUnit test is not practicable, an <<flow-integration-testing#,Integration Test>> should be added instead.
```

File: articles/contributing/flow-unit-testing.asciidoc [hunk 2]
```diff
@@ -35,10 +34,37 @@ It is always a good practice to refer to existing tests as an example to how to
...
+
+You can execute tests for single class with the following command:
+
+[source,terminal]
+----
+mvn -Dtest=<test-class-name> test -pl <module-folder-name>
+----
 
-For running all the unit tests in the project, you can execute `mvn test -am -pl flow`.
+Also, for running a single test inside a class:
```
Comment (dougjewsbury) [inline articles/contributing/flow-unit-testing.asciidoc:51]:
```suggestion
Also, to run a single test inside a class:
```

File: articles/contributing/flow-unit-testing.asciidoc [hunk 3]
```diff
@@ -35,10 +34,37 @@ It is always a good practice to refer to existing tests as an example to how to
...
+
+[source,terminal]
+----
+mvn -Dtest=<test-class-name>#<test-method-name> test -pl <module-folder-name>
+----
+
+For running all the unit tests in the project:
```
Comment (dougjewsbury) [inline articles/contributing/flow-unit-testing.asciidoc:58]:
```suggestion
To run all the unit tests in the project:
```

File: articles/contributing/flow-unit-testing.asciidoc [hunk 4]
```diff
@@ -35,10 +34,37 @@ It is always a good practice to refer to existing tests as an example to how to
...
+
+[source,terminal]
+----
+mvn test -am -pl flow
+----
 
 For running tests through your IDE, please refer to the documentation of your desired IDE.
```
Comment (dougjewsbury) [inline articles/contributing/flow-unit-testing.asciidoc:65]:
```suggestion
To run tests via your IDE, please refer to the IDE's documentation.
```

File: articles/contributing/flow-unit-testing.asciidoc [hunk 5]
```diff
@@ -35,10 +34,37 @@ It is always a good practice to refer to existing tests as an example to how to
 
 == How to Run the Tests
 
-Running all the tests `mvn verify -pl <module-name>` would run all the tests in the specified module.
+Running the following command results in running all the tests in the specified module:
 
-You can execute tests for single class by running the `mvn -Dtest=<test-class-name> test -pl <module-folder-name>`. Also, for running a single inside a class you can execute `mvn -Dtest=<test-class-name>#<test-method-name> test -pl <module-folder-name>`.
+[source,terminal]
+----
+mvn verify -pl <module-name>
```
Comment (caalador) [inline articles/contributing/flow-unit-testing.asciidoc:41]: This should probably mention that it works like this only for direct children of the directory as for instance running
`mvn test -pl flow-maven-plugin` in root will result in
```
[INFO] Scanning for projects...
[ERROR] [ERROR] Could not find the selected project in the reactor: flow-maven-plugin @ 
[ERROR] Could not find the selected project in the reactor: flow-maven-plugin -> [Help 1]
```
Where as running `mvn test -pl :flow-maven-plugin` will run the maven plugin module which is located at `./flow-plugins/flow-maven-plugin` optionally one could have the full path `mvn test -pl flow-plugins/flow-maven-plugin`

File: articles/contributing/flow-unit-testing.asciidoc [hunk 6]
```diff
@@ -35,10 +34,37 @@ It is always a good practice to refer to existing tests as an example to how to
...
+For running all the unit tests in the project:
+
+[source,terminal]
+----
+mvn test -am -pl flow
+----
 
 For running tests through your IDE, please refer to the documentation of your desired IDE.
+
+.Chrome Web Driver is required
```
Comment (caalador) [inline articles/contributing/flow-unit-testing.asciidoc:67]: If this is true then we should add the local-run profile to the client module pom to have it install the drivers automatically according to the `${project.rootdir}/drivers.xml` file

File: articles/contributing/flow-unit-testing.asciidoc (hunk 5 above)
Comment (taefi) [reply articles/contributing/flow-unit-testing.asciidoc]: Added a TIP to include details for running tests in nested modules.

**Review by caalador — APPROVED** [review]

**Pull request merged**